Android/app/src/main/java/com/example/kahye/common/network/RetrofitInstance.java

```diff
@@ -9,7 +9,8 @@
     private static Retrofit retrofit;
     private static OkHttpClient client;
     //URL can be easily changed
-    private static final String BASE_URL = "http://10.0.2.2:8000/api/v1/";
+    private static final String BASE_URL =
+            "http://52.8.187.167:8000/api/v1/";
```

Comment:
I think we can make another module that takes care of a network call. So, this file can import that module and pass "/api/v1" as a parameter and have that module make a network call.

Reply:
We don't understand exactly what you mean. What about talk about this in person?